include/octo-logger-cpp/context-info.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file context-info.hpp
+ * @author arad yaron (aradyaron98@gmail.com)
+ * @brief
+ * @version 0.1
+ * @date 2024-04-25
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+
+#ifndef CONTEXT_INFO_HPP_
+#define CONTEXT_INFO_HPP_
+
+#include <unordered_map>
+#include <string>
+#include <string_view>
+#include <sstream>
+
+
+namespace octo::logger
+{
+
+class ContextInfo
+{
+public:
+    typedef std::string_view ContextInfoKey;
+    typedef std::string ContextInfoValue;
+    typedef std::unordered_map<ContextInfoKey, ContextInfoValue> ContextInfoType;
+    typedef std::initializer_list<ContextInfoType::value_type> ContextInfoInitializerList;
+
+private:
+    ContextInfoType context_info_;
+
+public:
+    ContextInfo() = default;
```

Comment:
why do we need this if we have initializer constructor

Reply:
removed